src/locale/misc/views/en.yml

```diff
@@ -7,9 +7,11 @@ columnDialog:
   choices:
     firstAndLastName:
       description: First and last name of person
+      keywords: name combo
       title: First and Last Name
     personFields:
       description: Add your choice of fields like email, phone number, etc.
+      keywords: field multiple
```

Comment:
I think the keywords should contain the names of fields like email, country, zip code, etc. But I think that can be resolved later when we inevitably will want to review all of the text in this entire dialog to make sure it's consistent and communicative.